AndelaOSP/art-dashboard
# Bg Fetch list of assets when the user exits the assets details page

## What does this PR do?
Loads assets list from redux state when the user exits the assets details page

## Description of Task to be completed?
Same as ☝🏾

## How should this be manually tested?
Go to the assets list table and choose the very last page. On the last page, click any of the assets to see the details and click on the back arrow to go back to the assets list table. The list of assets for that page should be displayed

## What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162474002

## Any background context you want to add?
N/A

## Important notes
N/A

## Packages installed
None

## Deployment note

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
